docs/specifications/methods/undelegated-test.md

```diff
@@ -0,0 +1,44 @@
+# Undelegated tests
+
+## Background
+
+The purpose of an undelegated test is the test a future delegation before it is
+put in production.
+
```

Comment:
### Objective

The purpose of an undelegated test is the test a future delegation before it is put in production. Hence, while testing an undelegated test, when extra information (such as name servers, IP addresses) are provided as input